addons/yardage450/functions/fnc_turnOn.sqf

```diff
@@ -33,12 +33,15 @@ GVAR(active) = true;
 [{
     if (CBA_missionTime - GVAR(powerOnTime) > 30) exitWith {
         GVAR(active) = false;
-        74210 cutText ["", "PLAIN"];
+        QUOTE(ADDON) cutText ["", "PLAIN"];
         [_this select 1] call CBA_fnc_removePerFrameHandler;
     };
 
     if (currentWeapon ACE_player == "ACE_Yardage450" && cameraView == "GUNNER") then {
-        74210 cutRsc ["ACE_RscYardage450", "PLAIN", 1, false];
+        if (isNil {__dsp} || {isNull __dsp} || {ctrlIDD __dsp != -1}) then {
```

Comment:
I was going to suggest `isNil "__dsp"`, but realised that it's a macro...

All macros should be renamed to be upper case - I'll do it in another PR though.